### ISSUE TITLE
Add warning to prerequisites (SetupWizard)

### ISSUE THREAD

**Issue by verdulo** [microg/android_packages_apps_GmsCore#312]
My problem is not directly relate to microG project:
according to [prerequisites](https://github.com/microg/android_packages_apps_GmsCore/wiki/Prerequisites) I removed the 'SetupWizard' app, unfortunately after removing that app and restoring to factory settings I cannot use the home button, notifications' bar doesn't work, I cannot enter developer options - it's known 'bug' on Android 6. I think that users should be warned about that problem.

**Comment by ale5000-git** [comment]
I wonder if it would be better to keep SetupWizard.
I don't think it conflict with microG but I haven't tried.

**Comment by n76** [comment]
I don't believe there is an issue with the setup wizard that that is part of AOSP, but if Gapps has been installed it replaces the AOSP one with a different incompatible one.

**Comment by tomasz1986** [comment]
This bug has been around forever. At least ICS or even GB. It has nothing to do with Android 6 in particular. I never remove the SetupWizard when deleting GApps for this very reason.

**Comment by WinSuk** [comment]
This is not a bug on Android, but actually a feature. The setup wizard is ran on first boot, and sets a flag on the system saying the device has been set up and is ready to use.
With the Google setup wizard, that happens after it runs through adding your Google account, etc.
The AOSP version is a mostly-empty shell of an app that just sets the flag automatically. It's not visible, except maybe a blank screen for a few seconds.
With neither app installed, the flag never gets set, and the home button/status bar/etc remains disabled.

https://android.googlesource.com/platform/packages/apps/Provision/+/master/src/com/android/provision/DefaultActivity.java

**Comment by n76** [comment]
Not directly related but I think the work around could be the same: I had a time when doing a "dirty flash" for a ROM update got me stuck [with the same symptoms](http://retiredtechie.fitchfamily.org/2016/04/26/home-button-not-working-after-flashing-to-android-marshmallow/). My fix was:

```
> adb root
> adb shell
# settings --user 0 put secure user_setup_complete 1
```

**Comment by ale5000-git** [comment]
If micrG is installed then microG is detected as Google, so I think the SetupWizard of Google should work.
Does anyone tried?

**Comment by verdulo** [comment]
@ale5000-git I installed microG services as system user, after resetting to factory settings, the wizard gets stuck at select wi-fi screen.

**Comment by ale5000-git** [comment]
The wi-fi screen shouldn't have anything to do with microG, or I'm wrong?

**Comment by n76** [comment]
Correct, the wifi setup should be totally independent of microG.

**Comment by verdulo** [comment]
It shouldn't, but if you replace google services with microG (with spoofed signatures) some apparently unrelated things stop to work.

**Comment by motatuc** [comment]
I run into this problem just today which happened with Lineageos. For me the homescreen stood black and I could not do anything with the phone.

The people from Lineage rewrote the SetupWizard. Now this Wizard is looking for the google setup wizard because it sees gms.

**Comment by okeer** [comment]
I've got a FC loop with brand new LineageSetupWizard.
`adb shell rm -R /system/priv-app/LineageSetupWizard` solved the problem.
But this probably won't work after a factory reset.
